### ISSUE TITLE
Default value for Company field 

### ISSUE THREAD

**Issue by avielc** [grokability/snipe-it#10695]
### Snipe-IT Version

latest

### Operating System

linux

### Web Server

nginx

### PHP Version

7.4

### Is your feature request related to a problem? Please describe. A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]

Company field is a value I keep missing to check, even though I only have 1 company (the one I'm in)

### Describe the solution you'd like A clear and concise description of what you want to happen.

I think adding a checkbox (just like Status labels have a "default label" option) 
That would be just great!

### Additional context Add any other context or screenshots about the feature request here.

![image](https://user-images.githubusercontent.com/13572969/154569279-ace9f72f-2e95-47eb-8f80-c92e7dec7a65.png)
Basically, there's nothing really to add.

**Comment by snipe** [comment]
If you only have one company, why use the companies section at all? Should just be kinda implied that it's the company associated with the Snipe-IT install, no?

**Comment by avielc** [comment]
You're definitely correct. 
But just for order for the rare occasion a company may need some official record with information (e.,g. merge with another company, etc..)

**Comment by snipe** [comment]
If that's the case, just.. remember to add it? I'm not sure a default company is the solution here. If it's set by default, someone who forgot to *remove* a company value would make mistakes. At some point, people just have to remember to do stuff - tech can't save us from everything, and can create new problems.

**Comment by avielc** [comment]
that would make sense in some way. 
I guess I'll have to keep remembering or remove it all for the time being. 

Thanks!
